src/WebJobs.Script/Diagnostics/MetricEvent.cs

```diff
@@ -12,5 +12,7 @@ public abstract class MetricEvent
         public DateTime Timestamp { get; set; }
 
         public TimeSpan Duration { get; set; }
+
+        public string Data { get; set; }
```

Comment:
rename to language instead of data

Reply:
We want universal json field here to contain custom data. This will prevent us of creating new column for every new metric we want.
See @jeffhollan comment:
https://github.com/Azure/azure-functions-host/issues/3827